tests/conftest.py

```diff
@@ -27,7 +33,7 @@ def dummy(request: Any) -> ep.Tensor:
 
 
 def register(
-    backend: str, *, real: bool, attack: bool = True
+    backend: str, *, real: bool = False, attack: bool = True
```

Comment:
I am not sure I would make this change. Being explicit here seems better to me, but I am fine with it if you don't want to reverse this.